<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Chay Nabors</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>trtle</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>144</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>160</width>
     <height>144</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <addaction name="actionLoadFile"/>
    <addaction name="actionReset"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionTilesetViewer"/>
    <addaction name="actionBackgroundViewer"/>
    <addaction name="actionOAMViewer"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionFullscreen"/>
    <addaction name="actionSkipBootrom"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuDebug"/>
  </widget>
  <action name="actionTilesetViewer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tileset Viewer</string>
   </property>
  </action>
  <action name="actionBackgroundViewer">
   <property name="text">
    <string>Background Viewer</string>
   </property>
  </action>
  <action name="actionLoadFile">
   <property name="text">
    <string>Load Rom</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionOAMViewer">
   <property name="text">
    <string>OAM Viewer</string>
   </property>
  </action>
  <action name="actionSkipBootrom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skip Bootrom</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
